renovate.json

```diff
@@ -13,7 +13,7 @@
       "groupName": "UDS Package Kit Dependencies",
       "labels": ["app-deps"],
       "commitMessageTopic": "app-deps",
-      "matchDatasources": ["go"],
+      "matchManagers": ["gomod"],
```

Comment:
this is an attempt to fix triggering go mod tidy.
It clearly didn't work on the original renovate's PR, go.sum wasn't updated.